edweiss-firebase/functions/src/functions/lectures/likeQuestion.ts

```diff
@@ -1,13 +1,13 @@
 import LectureDisplay from 'model/lectures/lectureDoc';
 import { onAuthentifiedCall } from 'utils/firebase';
-import { CollectionOf, getDocumentAndRef } from 'utils/firestore';
+import { clean, CollectionOf, getDocumentAndRef } from 'utils/firestore';
 import { fail, ok } from 'utils/status';
 
-export const updateQuestion = onAuthentifiedCall(LectureDisplay.Functions.updateQuestion, async (userId, args) => {
-    if (!args.id)
+export const updateQuestion = onAuthentifiedCall(LectureDisplay.Functions.likeQuestion, async (userId, args) => {
```

Comment:
You might want to use `onSanitizedCall` to ensure full safety here. This function is vulnerable in its current state.